interface/frontend/src/components/common/OpenDICOM.vue

```diff
@@ -128,7 +129,7 @@
         if (_.isEmpty(dicom)) return ''
```

Comment:
Off-topic: I went and search for lodash usage and this is the only thing we're using it for?

https://github.com/concept-to-clinic/concept-to-clinic/search?utf8=%E2%9C%93&q=lodash&type=

Wonder if we should keep lodash :-? ... Most of its feature can be done with ES6

Overall, LGTM

Reply:
I will clean it up and push together with [these changes](https://github.com/concept-to-clinic/concept-to-clinic/issues/240#issuecomment-354178232), since that file will be changed a lot there.